CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Welcome to Lovelace Horizon Card contributing guide <!-- omit in toc -->
+
+Thank you for investing your time in contributing to our project! Any contribution you make will be reflected on the card and make lots of other users happy. :sparkles:.
+
+Please read [GitHub/docs Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+Use the table of contents icon <img src="/contributing/images/table-of-contents.png" width="25" height="25" /> on the top left corner of this document to get to a specific section of this guide quickly.
+
+## New contributor guide
+
+To get an overview of the project, read the [README](README.md). Here are some resources to help you get started with open source contributions:
+
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+
```

Comment:
It would be nice to have a section here on building the project. Mentioning that the project is built with yarn and not npm and a list of the available scripts:
- yarn build
- yarn test
- yarn lint
- yarn dev